src/components/forms/LoginFormModal.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import {
+  Button,
+  Dimmer,
+  Form,
+  Loader,
+  Modal,
+  TransitionablePortal,
+  Message,
+} from 'semantic-ui-react';
+import ModalFormHOC, { modalOptions } from '../hoc/ModalFormHOC';
+import LoginFields from './FormFieldComponents';
+
+export const LoginForm = ({
+  state, show, close, onSubmit, onChange,
+}) => {
+  const {
+    errors,
+    loading,
+    open,
+    user: { email, password },
+  } = state;
+  return (
+    <div>
+      <Link href="#" to="#" onClick={show}>
```

Comment:
You have  both "href" and "to" both pointing to #, since you are making use of the react-router Link, I suggest you remove the <a> tag attribute.

Reply:
Thanks for pointing this out, Like you suggested, If I remove the "href",  eslint will flag a violation error, notwithstanding, I will do some digging to look for a better alternative. cheers.